# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1370932880 -28800
#      Tue Jun 11 14:41:20 2013 +0800
# Node ID e15755a5a0d78ccf59f7eeb5b94c26df68b8a6ef
# Parent  804d2f7f50bf921346ac0f4c8460a5e6ea8b8043
Remove z-order assignments of invalid widgets.

* libgui/src/settings-dialog.ui: Remove the z-order assignments of widgets
that are no longer present.

diff --git a/libgui/src/settings-dialog.ui b/libgui/src/settings-dialog.ui
--- a/libgui/src/settings-dialog.ui
+++ b/libgui/src/settings-dialog.ui
@@ -1095,18 +1095,16 @@
            <size>
             <width>0</width>
             <height>81</height>
            </size>
           </property>
           <property name="title">
            <string>Storage Class Colors</string>
           </property>
-          <zorder>line_6</zorder>
-          <zorder>line_5</zorder>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_8">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>1</height>
@@ -1128,17 +1126,16 @@
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <zorder>verticalLayoutWidget_6</zorder>
-      <zorder>line_6</zorder>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Network</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
